.github/workflows/main.yml

```diff
@@ -0,0 +1,17 @@
+name: Aggregit
```

Comment:
The develop branch was behind main. That's why this commit is showing up here. Can you please merge develop back into your branch to remove Michael Jolley's commit?